app/src/main/kotlin/com/wire/android/ui/home/conversationslist/model/ConversationItem.kt

```diff
@@ -28,7 +29,7 @@ data class ConversationInfo(
 
 data class UserInfo(
     val avatarUrl: String = "",
-    val availabilityStatus: AvailabilityStatus = AvailabilityStatus.None
+    val availabilityStatus: UserStatus = UserStatus.NONE
```

Comment:
This `UserInfo` class seems like a copy of the `User` class. Any chance we can extract and unify both to a common one?

Reply:
I wasn't adding any of it, so not sure why is there separate 2 data classes.
My guess: both of them are just "holders" for the data needed to display on some screen (ConversationsList and Conversation). And in the future, when we'll have some `UserDataModel` that is fetched from the API, it will be mapped into these data classes in ViewModels. 
For now they are the same, but later it could change.